### DISCUSSION TITLE
How to group these folders group in a certain way but everything within the folder group alphabetically?

### DISCUSSION THREAD

**Discussion by TonyHoanTrinh** [storybookjs/storybook#15824] [Help]
Hello,

I currently have this kind of nesting with my stories.

![image](https://user-images.githubusercontent.com/26545361/129073472-0bae2272-1d44-4505-b9ff-1c39d5e114a3.png)

The stories within each of the folders such as Content, Getting Started, Guidelines and Patterns are sorted in alphabetically order.

Which is good, however though I want the folders themselves to be sorted in a custom order, 

Currently the order is Components, Content, Cypress, Getting Started, Guidelines, Patterns.

I want the order to be instead Getting Started, Guidelines, Patterns, Content, Components, Cypress

But also ensure that the contents within the folders are still sorted alphabetically.

Is this possible?

Thank you so much

**Comment by TonyHoanTrinh** [comment]
Please disregard, I figured out how to do this through this documentation (https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy)

**Comment by jonniebigodes** [comment]
@TonyHoanTrinh glad that you managed to solve your issue! That's great to hear. Let us know of any further issues/questions with Storybook.

Stay safe
